QuestionAnswering/QA_ICL/backend/README.md

```diff
@@ -0,0 +1,28 @@
+# Marie Backend
+
+## Architecture
+
+- [`fastapi_app`](./fastapi_app/) exposes APIs for:
+  - querying chemistry data via natural language
+  - searching for chemical species and zeolites based on some filtering criteria
+- [`triton_inference_server`](./triton_inference_server/) serves Sentence-BERT model.
+
+## Deployment with Docker (recommended)
+
+### Prerequisites
+
+- Docker
+
+### Steps
+
+1. Populate the required data and config parameters for the `fastapi_app` according to its README sections ['Required resources'](./fastapi_app/README.md#required-resources) and ['Configurable parameters'](./fastapi_app/README.md#configurable-parameters).
```

Comment:
Shouldn't it also be mentioned here to populate `./fastapi-variables.env`? Apologies if I missed it.

Reply:
This file should be removed because I have updated the app to use a YAML file for parameter configuration. I have removed this file in my latest commit.